module/VuFind/src/VuFind/Search/EDS/Options.php

```diff
@@ -7,6 +7,7 @@
  *
  * Copyright (C) EBSCO Industries 2013
  * Copyright (C) The National Library of Finland 2022
+ * Copyright (C) Villanova University 2023
```

Comment:
Your only change to this file is adding a copyright line. I think we can probably revert that.

Reply:
Oops, fixed.